common/src/Chat/Api/Http.purs

```diff
@@ -23,7 +29,38 @@ instance Show SignUpResponse where
   show = case _ of
     SignedUp -> "Signed Up"
     AlreadyRegistered -> "Already Registered"
-    
+
+data SignOutResponse
```

Comment:
I think we can respond `200 Ok` without body in case of signout as I don't see how is that information useful on the client.